source/_changelogs/5.4.0.md

```diff
@@ -37,3 +37,7 @@
 - Upgraded `electron` from `9.2.1` to `10.1.3`. Addressed in {% PR 8406 %}.
 - Upgraded `firefox-profile` from `2.0.0` to `4.0.0`. Addressed in {% PR 8786 %}.
 - Upgraded `node-forge` from `0.9.0` to `0.10.0`. Addressed in {% PR 8800 %}.
+
+**Documentation Changes:**
+
+- Added a new open source [visual testing](https://github.com/uktrade/cypress-image-diff) tool maintained by the british government.
```

Comment:
We only add doc changes to the Changelog relevant to our products release. 
```suggestion
```

Reply:
@jennifer-shehane thanks for your comment. I've updated the branch and also ensured the plugin meets all the requirements from the checklist above! let me know if you need anything else!